packages/rope/rope.0.5/opam

```diff
@@ -13,6 +13,7 @@ remove: [["ocamlfind" "remove" "rope"]]
 depends: [
   "ocamlfind"
   "ocamlbuild" {build}
+  "oasis" {build}
```

Comment:
Why is oasis added as a build dependency?

Reply:
Because it is used in setup.ml: https://github.com/Chris00/ocaml-rope/blob/master/setup.ml